src/views/Timer.vue

```diff
@@ -0,0 +1,209 @@
+            :i="item.i"
+            :key="item.i"
+            :w.sync="item.w"
+            :h.sync="item.h"
+            :x="item.x"
+            :y="item.y"
+            :container-width="props.containerWidth"
+            :row-height="props.rowHeight"
+            :is-draggable="true"
+            :is-resizable="false"
+            class-name="grid-item panel-header"
+            :cols="props.cols"
+            :height-from-children="true"
+            :max-rows="props.maxRows"
+            :canBeResizedWithAll="false"
+            handle=".header-panel"
+          >
+            <div ref="panelObserver" :id="item.i">
+              <div
+                :is="componentState[item.i].component"
+                v-bind="resolveProps(componentState[item.i].props)"
+              />
+            </div>
+          </vue-grid-item>
+        </template>
+      </vue-responsive-grid-layout>
+    </b-container>
+  </div>
+</template>
+
+<script>
+import { mapState, mapGetters } from 'vuex';
+import AlertPanel from '@/components/panels/AlertPanel.vue';
+import NewsPanel from '@/components/panels/NewsPanel.vue';
+import TimePanel from '@/components/panels/TimePanel.vue';
+import ResetPanel from '@/components/panels/ResetPanel.vue';
+import SortiePanel from '@/components/panels/SortiePanel.vue';
+import AcolytesPanel from '@/components/panels/AcolytesPanel.vue';
+import FissuresPanel from '@/components/panels/FissuresPanel.vue';
+import BountyPanel from '@/components/panels/BountyPanel.vue';
+import InvasionsPanel from '@/components/panels/InvasionsPanel.vue';
+import EventsPanel from '@/components/panels/EventsPanel.vue';
+import DarvoDealsPanel from '@/components/panels/DarvoDealsPanel.vue';
+import SalesPanel from '@/components/panels/SalesPanel.vue';
+import VoidTraderPanel from '@/components/panels/VoidTraderPanel.vue';
+import NightwavePanel from '@/components/panels/NightwavePanel.vue';
+
+export default {
+  name: 'timers',
+  components: {
+    AlertPanel,
+    NewsPanel,
+    TimePanel,
+    ResetPanel,
+    SortiePanel,
+    AcolytesPanel,
+    FissuresPanel,
+    BountyPanel,
+    InvasionsPanel,
+    EventsPanel,
+    DarvoDealsPanel,
+    SalesPanel,
+    VoidTraderPanel,
+    NightwavePanel,
+  },
+  data() {
+    return {
+      components: {},
+      breakpoint: 'md',
+      cols: 2,
+      breakpoints: { lg: 1800, md: 1200, sm: 996, xs: 768, xxs: 480 },
+      colsAll: { lg: 3, md: 3, sm: 2, xs: 1, xxs: 1 },
+      isDraggable: true,
+      isResizable: true,
+      lastUpdate: 0
+    };
+  },
+  mounted() {
+    this.components = this.gridState.components;
+    this.lastUpdate = Date.now();
+  },
+  methods: {
+    track() {
+      this.$ga.page('/');
+    },
+    updateComponents(layout, breakpoint) {
+      layout.forEach((itemSize) => {
+        this.$set(this.components[itemSize.i], breakpoint, itemSize);
+      });
+      const currTime = Date.now();
+      if (currTime - this.lastUpdate > 500) {
```

Comment:
Not saying it should change, but I just want to remark that this TTL is a bit strange, as I'm used to seeing them in multiples of a minute.

Reply:
It is, but it's worked fine thusfar.